src/database/database.js

```diff
@@ -2,6 +2,7 @@ const strings = require('./../utils/strings');
 const configstore = require('conf');
 const mongoDb = require('./mongoDb/mongoDb');
 const mysql = require('./mysql/mysql');
+const postgresql = require('./postgresql/postgrsql');
```

Comment:
Typo

Reply:
Is this resolved?

Reply:
Do test the application before committing.

Reply:
ohhh I am doing it right now